### PR TITLE
Popover issue fix

### LINKED ISSUE #119: Do not show popover when a button in the BotsTable is hovered

### PR DESCRIPTION
**Pull request by react-developer-ssa-group**
Closes #119

### CODE REVIEW

File: packages/core/src/components/Popover/hooks/usePopover.tsx [hunk 1]
```diff
@@ -49,29 +49,35 @@ export const usePopover: UsePopover = ({
...
-  const hover = useHover(context, {
-    enabled: controlledOpen == null,
-    handleClose: safePolygon(),
-  });
 
   const interactionsHooks = [dismiss, role];
 
   switch (interactionsEnabled) {
     case 'click': {
+      const click = useClick(context, {
```
Comment (arthur-kashaiev-ssa-group) [inline packages/core/src/components/Popover/hooks/usePopover.tsx:59]: image
https://react.dev/warnings/invalid-hook-call-warning

**Review by arthur-kashaiev-ssa-group — CHANGES_REQUESTED** [review]

File: packages/core/src/components/Popover/hooks/usePopover.tsx (hunk 1 above)
Comment (react-developer-ssa-group) [reply packages/core/src/components/Popover/hooks/usePopover.tsx]: Fixed

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 8       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 1  | ❗       |
    


<details>
<summary>🟡 storybook/components-popover--interactions-enabled-on-hover__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-popover--interactions-enabled-on-hover__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-popover--interactions-enabled-on-hover__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-popover--interactions-enabled-on-hover__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-popover--interactions-enabled-on-hover-and-click__[w1440px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-popover--interactions-enabled-on-hover-and-click__[w1920px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-popover--interactions-enabled-on-hover-and-click__[w390px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



<details>
<summary>🟡 storybook/components-popover--interactions-enabled-on-hover-and-click__[w900px] (click to expand)</summary>

| Image |
| --- |
|  |
</details>



*❗ storybook/utility-responsive-image--responsive-image__[w1440px]*: 


| Before | After |
| --- | --- |
|  |  |

**Comment by lost-pixel-signals** [comment]
| Shot Type       | Count                | Status   |
| -------------   | -------------------- | -------- |
| Added           | 0       | 🟡       |
| Deleted         | 0     | ❌       |
| Regression      | 4  | ❗       |
    


*❗ custom/widgets-collapsiblenavbar-opened__[w899px]_menu_opened*: 


| Before | After |
| --- | --- |
|  |  |



*❗ custom/widgets-user-profile--is-opened__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/widgets-user-profile--default__[w1920px]*: 


| Before | After |
| --- | --- |
|  |  |



*❗ storybook/widgets-user-profile--default__[w390px]*: 


| Before | After |
| --- | --- |
|  |  |

**Pull request merged**